internal/api/status.go

```diff
@@ -24,15 +25,18 @@ func status(config *RouterConfig) API {
 		ContentType = "application/json"
 	)
 	type Response struct {
-		Version string        `json:"version"`
-		OS      string        `json:"os"`
-		Arch    string        `json:"arch"`
-		UpTime  time.Duration `json:"uptime"`
-
-		CPUs       int    `json:"num_cpu"`
-		UsableCPUs int    `json:"num_cpu_used"`
-		HeapAlloc  uint64 `json:"mem_heap_used"`
-		StackAlloc uint64 `json:"mem_stack_used"`
+		Version    string        `json:"version"`
+		OS         string        `json:"os"`
+		Arch       string        `json:"arch"`
+		UpTime     time.Duration `json:"uptime"`
+		CPUs       int           `json:"num_cpu"`
+		UsableCPUs int           `json:"num_cpu_used"`
+		HeapAlloc  uint64        `json:"mem_heap_used"`
+		StackAlloc uint64        `json:"mem_stack_used"`
+
+		KeyStoreLatency     time.Duration `json:"keystore_latency,omitempty"`
+		KeyStoreUnavailable bool          `json:"keystore_unavailable,omitempty"`
+		KeyStoreUnreachable bool          `json:"keystore_unreachable,omitempty"`
```

Comment:
Booleans shouldn't have negatives in the names, when avoidable, since it hurts readability:

```suggestion
		KeyStoreAvailable bool            `json:"keystore_available"`
		KeyStoreReachable bool            `json:"keystore_reachable"`
```

`if a && b || !KeyStoreUnavailable` is hard to logically parse. 

`KeyStoreAvailable` and `!KeyStoreAvailable` is easier to parse since neither have double negatives.